src/main/java/org/terasology/managerInterface/nui/ManagerInterfaceHUDElement.java

```diff
@@ -244,25 +221,7 @@ public void populateCreaturesMenu() {
         }
 
         EntityManager entityManager = CoreRegistry.get(EntityManager.class);
```

Comment:
Note that all references to CoreRegistry.get(class) need to be replaced with either @In annotations (best idea) or context.get(class) codes (find-and-replace).  You'll probably need an @In annotation to pull in a context.